scripts/is/intg/helper-scripts/provision_db_is.sh

```
@@ -0,0 +1,213 @@
+# expressly. You may not alter or remove any copyright or other notice from
+# copies of this content.
+#
+# ------------------------------------------------------------------------
+
+#Running Database scripts for WSO2-IS
+echo "Running DB scripts for WSO2-IS..."
+
+#Define parameter values for Database Engine and Version
+
+DB_ENGINE='CF_DBMS_NAME'
+DB_ENGINE_VERSION='CF_DBMS_VERSION'
```

Shellcheck (0.11.0)

[warning] DB_ENGINE_VERSION appears unused. Verify use (or export if used externally).

(SC2034)